sentry-android/src/main/java/com/joshdholtz/sentry/Sentry.java

```diff
@@ -941,42 +950,42 @@ public static JSONObject getStackTrace(Throwable t) throws JSONException {
             JSONArray frameList = new JSONArray();
 
             for (StackTraceElement ste : t.getStackTrace()) {
-                JSONObject frame = new JSONObject();
-
-                String method = ste.getMethodName();
-                if (method.length() != 0) {
-                    frame.put("function", method);
-                }
+                frameList.put(frameJson(ste));
+            }
 
-                int lineno = ste.getLineNumber();
-                if (!ste.isNativeMethod() && lineno >= 0) {
-                    frame.put("lineno", lineno);
-                }
+            JSONObject frameHash = new JSONObject();
+            frameHash.put("frames", frameList);
 
-                boolean inApp = true;
+            return frameHash;
+        }
 
-                String className = ste.getClassName();
-                frame.put("module", className);
+		// Convert a StackTraceElement to a sentry.interfaces.stacktrace.Stacktrace JSON object.
+		static JSONObject frameJson(StackTraceElement ste) throws JSONException {
+			final JSONObject frame = new JSONObject();
 
-                // Take out some of the system packages to improve the exception folding on the sentry server
-                if (className.startsWith("android.")
-                        || className.startsWith("java.")
-                        || className.startsWith("dalvik.")
-                        || className.startsWith("com.android.")) {
+			final String method = ste.getMethodName();
+			if (method.length() != 0) {
+				frame.put("function", method);
+			}
 
-                    inApp = false;
-                        }
+			final String fileName = ste.getFileName();
+			if (fileName.length() > 0) {
+				frame.put("filename", fileName);
+			}
 
-                frame.put("in_app", inApp);
+			int lineno = ste.getLineNumber();
+			if (!ste.isNativeMethod() && lineno >= 0) {
+				frame.put("lineno", lineno);
+			}
 
-                frameList.put(frame);
-            }
+			String className = ste.getClassName();
```

Comment:
Looks like Sentry.java has a mix of spaces and tabs in different parts of the file.